ShenkinStore/Models/User.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ShenkinStore.Models
+{
+    public class User
```

Comment:
Please add validations to all the fields

Reply:
**Related to the upper comment**